openwrt-addons/etc/kalua_init

```diff
@@ -98,8 +98,10 @@ mv "$LOADER" "$LOADER_FINAL"
 }
 
 logger -s -- "$0: [OK] generated '$LOADER_ENTRY' using files in '$POOLDIR'"
-case "$PS1" in
-	*'@'*)	# interactive shell (set from /etc/profile)
+case "$PS1-display$DISPLAY" in
```

Comment:
`$DISPLAY` ist eigentlich so gut wie nie in einer SSH-Session gesetzt, wenn man sich von Router zu Router hangelt ist das definitiv nicht da. Das wird nur bei X11 Forwarding gesetzt oder eben lokal unter X. 

Oder hab ich da was falsch verstanden?